python-backend/app/api/mines/subscription/models/subscription.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+from app.api.utils.include.user_info import User
+from sqlalchemy.schema import FetchedValue
+from sqlalchemy.dialects.postgresql import UUID
+
+from app.extensions import db
+from app.api.utils.models_mixins import  Base
+from werkzeug.exceptions import BadRequest, NotFound
+
+
+class Subscription( Base):
+    __tablename__ = 'subscription'
+    user_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    mine_guid = db.Column(UUID(as_uuid=True),db.ForeignKey('mine.mine_guid'))
+    idir = db.Column(db.String, nullable=False)
+
```

Comment:
this is becomes a join table between mines and our new core_users table, then this model could change significantly, i'm not sure if we want to change this to use that information yet.

Reply:
I'm good with waiting on changes before refactoring again. :P